### DISCUSSION TITLE
What do you like seeing in DemoDays?

### DISCUSSION THREAD

**Discussion by javabster** [Qiskit/.github#5] [DemoDay]
Over the past few DemoDays we have seen a wide range of types of demos, including:

- qiskit performance improvements,
- work in progress features/PRs,
- fixes/features from external contributors,
- intern project demos

among others. We'd like to know what you've enjoyed the most and what you would like to see more of. Or if there's anything we haven't done yet that you would like to suggest tell us that too!

For reference, you can see details of past and future DemoDays in the wiki [here](https://github.com/qiskit-community/feedback/wiki/Qiskit-DemoDays)

**Comment by 1ucian0** [comment]
I particularly enjoy performance improvements, like the demo from @jakelishman https://github.com/qiskit-community/feedback/wiki/Qiskit-DemoDays#sep-30-2021 .

I think small, specific, and self-contain is very much align with the demoday spirit and performance improvements have a good chance to hit there.
